### ISSUE TITLE
Cache the OAuth2 access token

### ISSUE THREAD

**Issue by mgechev** [guess-js/guess#12]
The `GuessPlugin` should cache the access token so it doesn't ask the user for authorization on each run.

**Comment by dijs** [comment]
Agreed. Also being able to pass the access token with an environment variable would be helpful for CI builds.

**Comment by mgechev** [comment]
This is already supported, and probably not documented too well. Check out [this](https://github.com/guess-js/guess/blob/master/packages/guess-webpack/src/ga-provider.ts#L16) property.

**Comment by wardpeet** [comment]
@dijs you can find an example here:
https://github.com/gatsbyjs/gatsby/blob/master/www/gatsby-config.js#L17-L28

**Comment by dijs** [comment]
Thank you very much.
